jstz_api/src/global.rs

```diff
@@ -28,44 +28,41 @@ impl TextEncoder {
     }
 }
 
-pub struct TextEncoderApi;
-impl TextEncoderApi {
-    const NAME: &str = "TextEncoder";
+pub struct GlobalApi;
+impl GlobalApi {
     fn atob(_: &JsValue, args: &[JsValue], _: &mut Context) -> JsResult<JsValue> {
         let data = args
             .get_or_undefined(0)
             .as_string()
             .ok_or_else(|| JsNativeError::typ().with_message("expected string"))?;
```

Comment:
Please use `try_js_into`

Reply:
```
lesenr1:[~/work/jstz]:
> cargo build
   Compiling jstz_api v0.1.0 (/home/lesenr1/work/jstz/jstz_api)
error[E0277]: the trait bound `&JsString: TryFromJs` is not satisfied
  --> jstz_api/src/global.rs:34:45
   |
34 |         let data = args.get_or_undefined(0).try_js_into(context)?;
   |                                             ^^^^^^^^^^^ the trait `TryFromJs` is not implemented for `&JsString`
 ```

Reply:
This is because the inferred type for `data` is `&JsString`, but if you annotate `data` with `JsString`, then this will work just fine.

Reply:
done

Reply:
same thing
```
error[E0277]: the trait bound `JsString: TryFromJs` is not satisfied
  --> jstz_api/src/encoding/global.rs:34:55
   |
34 |         let data: JsString = args.get_or_undefined(0).try_js_into(context)?;
   |                                                       ^^^^^^^^^^^ the trait `TryFromJs` is not implemented for `JsString`
   ```

Reply:
done